src/appManager.js

```diff
@@ -1,20 +1,26 @@
+var myApp = {};
```

Comment:
Have I ever told you the story of how terribly annoying I find variable names that start with `my`? :P No action item required, really, but this is just one of my little pet peeves - I couldn't just let it slide. :)

Oh, and just for the sake of argumentation: is it really _your_ app when it hits the general public? Or when other people start developing on it alongside you? Would you really be that selfish as to deprive the rest of us from the joys and wonders of owning even but a small piece of _your_ app? :P